<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keyboard</class>
 <widget class="QDialog" name="keyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backbtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnX">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn9">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="okbtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelbtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn0">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnwell">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnspace">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>160</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btn0</tabstop>
  <tabstop>btn1</tabstop>
  <tabstop>btn2</tabstop>
  <tabstop>btn3</tabstop>
  <tabstop>btn4</tabstop>
  <tabstop>btn5</tabstop>
  <tabstop>btn6</tabstop>
  <tabstop>btn7</tabstop>
  <tabstop>btn8</tabstop>
  <tabstop>btn9</tabstop>
  <tabstop>btnX</tabstop>
  <tabstop>btnwell</tabstop>
  <tabstop>okbtn</tabstop>
  <tabstop>cancelbtn</tabstop>
  <tabstop>backbtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
